applications/MeshingApplication/CMakeLists.txt

```
@@ -59,6 +59,40 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 add_library(KratosMeshingCore SHARED ${KRATOS_MESHING_APPLICATION_CORE} ${KRATOS_MESHING_APPLICATION_CORE_MMG} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
 
 if(${INCLUDE_MMG} MATCHES ON)
+    # Define includes and libraries with just one include
+    IF (MMG_ROOT)
+        SET(MMG_INCLUDE_DIR ${MMG_ROOT}/include/mmg/)
```

@loumalouomega I need to set this like this in order for it to work:
```suggestion
        SET(MMG_INCLUDE_DIR ${MMG_ROOT}/include/)
```

In the old way I had the following (note there's no final  `/mmg`, as it is done in the [wiki](https://github.com/KratosMultiphysics/Kratos/wiki/%5BUtilities%5D-MMG-Process#how-can-i-install-this-library) ):

`-DMMG_INCLUDE_DIR="~/src/mmg/build/include"`

My installation looks like this:

```
marc@marc-VM:~/src/mmg/build$ ls include/
mmg
marc@marc-VM:~/src/mmg/build$ ls include/mmg/
libmmgf.h  libmmg.h  mmg2d  mmg3d  mmgs
marc@marc-VM:~/src/mmg/build$ 
```

I guess this has to do with the way mmg was compiled.    My installation uses the configure file in:

https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh


While the CI does it like this:
https://github.com/KratosMultiphysics/Kratos/blob/1b9fdd42ce3cf8cc47f77ac8619099952a5e5ad2/scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile#L50-L55

Any idea on what to do here?

---

I suggest adding the following:

```
        if(NOT DEFINED MMG_INCLUDE_DIR)
            SET(MMG_INCLUDE_DIR ${MMG_ROOT}/include/mmg)
        endif(NOT DEFINED MMG_INCLUDE_DIR)
```

So that the users can specify the correct include_dir

---

OK

---

Ok, just removing the "mmg/" part seems to work for the CI as well, we can ignore my second message